worldengine/simulations/temperature.py

```diff
@@ -27,13 +25,18 @@ def execute(self, world, seed):
         world.set_temperature(t, t_th)
 
     @staticmethod
-    def _calculate(world, seed, elevation, mountain_level):
+    def _calculate(world, seed):
         width = world.width
         height = world.height
+        e = world.elevation['data']
+        mountain_level = world.start_mountain_th()  # returns minimum elevation value for mountains (i.e. hills)
+        highest_mountain_level = e.max() # world.level_of_mountain(numpy.unravel_index(e.argmax(), e.shape))  # height of the highest point relative to mountain_level
 
         rng = numpy.random.RandomState(seed)  # create our own random generator
         base = rng.randint(0, 4096)
         temp = numpy.zeros((height, width), dtype=float)
+        
+        min_altitude_factor = 0.3  # used for the top of the highest mountain; must be in [0, 1]
```

Comment:
This factor makes the highest mountain tops pretty cold, the effects can be seen in [the temperature-map](https://github.com/Mindwerks/worldengine-data/pull/17/files). It is mostly a guess, consolidated by Googling the average temperature in Antarctica and on the Mount Everest.